src/ui/public/vis_maps/visualizations/marker_types/base_marker.js

```diff
@@ -218,7 +218,12 @@ export default function MarkerFactory() {
      * @return undefined
      */
     _showTooltip(feature, latLng) {
-      if (!this.map) return;
+      if (!this.map) {
+        return;
+      }
+      if (!this._tooltipFormatter) {
```

Comment:
i think something like 
`
const hasMap = !!this.map;
const hasTooltipFormatter = !!this._tooltipFormatter);
if (!hasMap || !hasTooltipFormatter) return;
`